docs/content/user-guide/inference.md

```
@@ -0,0 +1,235 @@
+            \"content\": \"Tell me a short story\"
+          }
+        ],
+        \"max_tokens\": 200,
+        \"stream\": true
+      }" \
+  "${MODEL_URL}/v1/chat/completions"
+```
+
+**Example streaming response:**
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)